resources/test_data/sqlite_testrunner_queries.sql

```diff
@@ -287,7 +287,7 @@ sELEcT Sum(b + b) AS sum_b_b from mixed;
 
 -- Aggregates with NULL
 SELECT a, MAX(b) FROM mixed_null GROUP BY a;
-SELECT a, MAX(b) FROM mixed_null GROUP BY a ORDER BY MAX(b) DESC;
+SELECT a, MAX(b) FROM mixed_null GROUP BY a ORDER BY MAX(b), a DESC;
```

Comment:
Hm?

Reply:
If the query contains "ORDER BY", we expect the results to match line-by-line. However, the output order of GROUP BY is undefined, and the new libc++ changed it.